package.json

```diff
@@ -26,9 +26,10 @@
     "electron-build": "electron-builder --mac --windows --linux",
     "electron-dev": "concurrently \"BROWSER=none yarn start\" \"wait-on http://localhost:3000 && electron .\"",
     "format:staged": "lint-staged",
+    "generate-types": "typechain --target=web3-v1 --outDir './src/types/contracts' './node_modules/@gnosis.pm/safe-contracts/build/contracts/*.json'",
```

Comment:
can we call it `generate-contract-types`?

Reply:
this is still a `wip` name, in future I thought about something `generate-types:contracts`, `generate-types:tokens`, etc

Reply:
but yeah makes sense to do it now